src-tauri/src/node.rs

```diff
@@ -148,63 +173,66 @@ fn create_configuration<CS: ChainSpec + 'static>(
+            },
+            rpc_http: None,
+            rpc_ws: Some("127.0.0.1:9944".parse().expect("IP and port are valid")),
+            rpc_ipc: None,
+            rpc_methods: RpcMethods::Safe,
+            rpc_ws_max_connections: None,
+            // Below CORS are default from Substrate
+            rpc_cors: Some(vec![
+                "http://localhost:*".to_string(),
+                "http://127.0.0.1:*".to_string(),
+                "https://localhost:*".to_string(),
+                "https://127.0.0.1:*".to_string(),
+                "https://polkadot.js.org".to_string(),
+                "tauri://localhost".to_string(),
+            ]),
+            rpc_max_payload: None,
+            ws_max_out_buffer_capacity: None,
+            prometheus_config: None,
+            telemetry_endpoints,
+            default_heap_pages: None,
+            offchain_worker: OffchainWorkerConfig::default(),
+            force_authoring: env::var("FORCE_AUTHORING")
+                .map(|force_authoring| force_authoring.as_str() == "1")
+                .unwrap_or_default(),
+            disable_grandpa: false,
+            dev_key_seed: None,
+            tracing_targets: None,
+            tracing_receiver: TracingReceiver::Log,
+            chain_spec: Box::new(chain_spec),
+            max_runtime_instances: 8,
+            announce_block: true,
+            role,
+            base_path: Some(base_path),
+            informant_output_format: Default::default(),
+            runtime_cache_size: 2,
         },
-        rpc_http: None,
-        rpc_ws: Some("127.0.0.1:9944".parse().expect("IP and port are valid")),
-        rpc_ipc: None,
-        rpc_methods: RpcMethods::Safe,
-        rpc_ws_max_connections: None,
-        // Below CORS are default from Substrate
-        rpc_cors: Some(vec![
-            "http://localhost:*".to_string(),
-            "http://127.0.0.1:*".to_string(),
-            "https://localhost:*".to_string(),
-            "https://127.0.0.1:*".to_string(),
-            "https://polkadot.js.org".to_string(),
-            "tauri://localhost".to_string(),
-        ]),
-        rpc_max_payload: None,
-        ws_max_out_buffer_capacity: None,
-        prometheus_config: None,
-        telemetry_endpoints,
-        default_heap_pages: None,
-        offchain_worker: OffchainWorkerConfig::default(),
-        force_authoring: env::var("FORCE_AUTHORING")
-            .map(|force_authoring| force_authoring.as_str() == "1")
-            .unwrap_or_default(),
-        disable_grandpa: false,
-        dev_key_seed: None,
-        tracing_targets: None,
-        tracing_receiver: TracingReceiver::Log,
-        chain_spec: Box::new(chain_spec),
-        max_runtime_instances: 8,
-        announce_block: true,
-        role,
-        base_path: Some(base_path),
-        informant_output_format: Default::default(),
-        runtime_cache_size: 2,
+        force_new_slot_notifications: false,
```

Comment:
Nazar, I don't not 100% sure this should be false/true. Can you double check?

Reply:
forgot to tag properly in the previous comment -> @nazar-pc

Reply:
`false` is fine